<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>509</width>
    <height>648</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HoneyPipe</string>
  </property>
  <property name="windowOpacity">
   <double>0.920000000000000</double>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QComboBox" name="cbInt">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>171</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblInt">
    <property name="geometry">
     <rect>
      <x>76</x>
      <y>10</y>
      <width>121</width>
      <height>10</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose interface</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblIp">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>10</y>
      <width>70</width>
      <height>10</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose IP</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbIp">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>30</y>
      <width>171</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>-93</x>
      <y>70</y>
      <width>801</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>-100</x>
      <y>490</y>
      <width>761</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>471</width>
      <height>411</height>
     </rect>
    </property>
    <property name="title">
     <string>Scanner</string>
    </property>
    <widget class="QPushButton" name="btnScan">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>17</y>
       <width>98</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Scan</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>298</x>
       <y>23</y>
       <width>111</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>ms (timeout/ip)</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="twMain">
     <property name="geometry">
      <rect>
       <x>8</x>
       <y>60</y>
       <width>452</width>
       <height>241</height>
      </rect>
     </property>
     <property name="baseSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="rowCount">
      <number>0</number>
     </property>
     <property name="columnCount">
      <number>2</number>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>100</number>
     </attribute>
     <column>
      <property name="text">
       <string>IP</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>MAC</string>
      </property>
     </column>
    </widget>
    <widget class="QSpinBox" name="spinBox">
     <property name="geometry">
      <rect>
       <x>227</x>
       <y>18</y>
       <width>59</width>
       <height>27</height>
      </rect>
     </property>
     <property name="minimum">
      <number>5</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>30</number>
     </property>
    </widget>
    <widget class="QPushButton" name="pbSetDest">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>310</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Set</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="leDestMAC">
     <property name="geometry">
      <rect>
       <x>324</x>
       <y>365</y>
       <width>141</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pbSetSource">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>310</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Set</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>342</y>
       <width>111</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Source Victim :</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="leSourceMAC">
     <property name="geometry">
      <rect>
       <x>14</x>
       <y>365</y>
       <width>141</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>343</y>
       <width>191</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Dest Victim (usually router):</string>
     </property>
    </widget>
    <zorder>btnScan</zorder>
    <zorder>btnScan</zorder>
    <zorder>label</zorder>
    <zorder>twMain</zorder>
    <zorder>spinBox</zorder>
    <zorder>pbSetDest</zorder>
    <zorder>leDestMAC</zorder>
    <zorder>pbSetSource</zorder>
    <zorder>label_2</zorder>
    <zorder>leSourceMAC</zorder>
    <zorder>label_3</zorder>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>500</y>
      <width>461</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>Spoof and Attack</string>
    </property>
    <widget class="QPushButton" name="pbSpoof">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>131</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Start spoofing</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>20</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Protocol ?</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbProtocol">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>40</y>
       <width>121</width>
       <height>27</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Netsoul</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="pbPlay">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>30</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Play</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>509</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="sbMain">
   <property name="toolTip">
    <string/>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
